com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -338,7 +363,7 @@ SensorComponent[] sensorComponents
             }
 
             // If the model uses discrete control but does not have an input for action masks
-            if (isContinuous == ModelActionType.Discrete)
+            if (modelActionType == ModelActionType.Discrete || modelActionType == ModelActionType.Hybrid)
```

Comment:
I think some of this code would be cleaner if ModelActionType were a Flags instead of Enum.